### PR TITLE
[ACCEL] Fix instructor CSV files

### PR DESCRIPTION
**Pull request by arsentur**
**Description:**
Add `S3_USE_SIGV4` and `S3_HOST` settings to `lms/envs/aws.py`.
Add a condition to DjangoStorageReport and required settings for "s3" storage.

**Youtrack:**
https://youtrack.raccoongang.com/issue/ACCEL-49

**Settings:**

**lms.auth.json**
```
{
    ...
    "AWS_ACCESS_KEY_ID": "<AWS_ACCESS_KEY_ID>",
    "AWS_QUERYSTRING_AUTH": false,
    "AWS_S3_CUSTOM_DOMAIN": "<AWS_S3_CUSTOM_DOMAIN>",
    "AWS_SECRET_ACCESS_KEY": "<AWS_SECRET_ACCESS_KEY>",
    "AWS_STORAGE_BUCKET_NAME": "<AWS_STORAGE_BUCKET_NAME>",
    ...
}
```

**lms.env.json**
```
{
  ...
  "GRADES_DOWNLOAD": {
        "BUCKET": "<bucket>",
        "ROOT_PATH": "<path>",
        "STORAGE_CLASS": "storages.backends.s3boto.S3BotoStorage",
        "STORAGE_KWARGS": {
            "location": "<location>"
        },
        "STORAGE_TYPE": "s3",
        "CUSTOM_DOMAIN": "<custom_domain>"
    },
   ...
    "S3_HOST": "<s3_host>",
    "S3_USE_SIGV4": true,
}
```

### CODE REVIEW

File: lms/envs/aws.py [hunk 1]
```diff
@@ -517,6 +517,12 @@
 AWS_QUERYSTRING_AUTH = AUTH_TOKENS.get('AWS_QUERYSTRING_AUTH', True)
 AWS_S3_CUSTOM_DOMAIN = AUTH_TOKENS.get('AWS_S3_CUSTOM_DOMAIN', 'edxuploads.s3.amazonaws.com')
 
+S3_USE_SIGV4 = ENV_TOKENS.get('S3_USE_SIGV4', False)
+S3_HOST = ENV_TOKENS.get('S3_HOST', None)
```
Comment (idegtiarov) [inline lms/envs/aws.py]: None is default value which could be omitted.

File: lms/djangoapps/instructor_task/models.py [hunk 2]
```diff
@@ -198,16 +198,21 @@ def from_config(cls, config_name):
...
-                storage_kwargs={
-                    'bucket': config['BUCKET'],
-                    'location': config['ROOT_PATH'],
-                    'custom_domain': config.get("CUSTOM_DOMAIN", None),
-                    'querystring_expire': 300,
-                    'gzip': True,
-                },
-            )
+            if (
+                    hasattr(settings, 'S3_HOST') and settings.S3_HOST and
```
Comment (idegtiarov) [inline lms/djangoapps/instructor_task/models.py]: Why do we need to test `hasattr` if it is added in current PR?

File: lms/envs/aws.py (hunk 1 above)
Comment (arsentur) [reply lms/envs/aws.py]: Done

File: lms/djangoapps/instructor_task/models.py (hunk 2 above)
Comment (arsentur) [reply lms/djangoapps/instructor_task/models.py]: Done

**Pull request merged**